tests/install_testbed/create_resources.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -ex
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FIWARE project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+# Author: Chema
+
+. config_vars
+
+neutron net-create ext-net --router:external True --provider:physical_network external --provider:network_type flat
```

Comment:
Just for checking. Usually the FIWARE Lab nodes are using the node-int-net-## for the external network, is there any problem in the skuld if the network is ext-net?